components/Activity/HeroCarousel.module.less

```
@@ -0,0 +1,170 @@
+  background: #050816;
+  min-height: calc(100vh - var(--hero-carousel-offset) - 2.5rem);
+}
+
+.carousel,
+.item,
+.slideCard {
+  min-height: inherit;
+}
+
+.carousel {
+  :global(.carousel-inner) {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)